client/src/Visit.test.tsx

```diff
@@ -93,93 +129,72 @@ describe('Visit', () => {
     expect(genericErrors.length).toBe(1);
   });
 
-  it('renders a generic error when token throws an error', async () => {
-    const fetchTokenSpy = jest.spyOn(FetchToken, 'fetchToken');
-    fetchTokenSpy.mockImplementation(
-      async (): Promise<CommunicationUserToken> => {
-        throw new Error('test error');
-      }
-    );
-
+  it('should render JoinTeamsMeeting when config is loaded and meeting link is not set', async () => {
     const visit = mount(<Visit />);
 
     await runFakeTimers();
 
     visit.update();
 
     const spinners = visit.find(Spinner);
-    const genericErrors = visit.find(GenericError);
+    const joinMeetings = visit.find(JoinTeamsMeeting);
 
     expect(spinners.length).toBe(0);
-    expect(genericErrors.length).toBe(1);
+    expect(joinMeetings.length).toBe(1);
   });
 
-  it('should render JoinTeamsMeeting when config and token are loaded and meeting link is not set', async () => {
-    const fetchConfigSpy = jest.spyOn(FetchConfig, 'fetchConfig');
-    fetchConfigSpy.mockReturnValue(
-      Promise.resolve({
-        communicationEndpoint: 'endpoint=test_endpoint;',
-        microsoftBookingsUrl: '',
-        chatEnabled: true,
-        screenShareEnabled: true,
-        companyName: '',
-        theme: generateTheme('#FFFFFF'),
-        waitingTitle: '',
-        waitingSubtitle: '',
-        logoUrl: ''
-      } as AppConfigModel)
-    );
+  it('should render TeamsMeeting when config is loaded and meeting link is a teams meeting link', async () => {
+    const getChatThreadIdFromTeamsLinkSpy = jest.spyOn(GetMeetingLink, 'getChatThreadIdFromTeamsLink');
+    getChatThreadIdFromTeamsLinkSpy.mockReturnValue('threadId');
 
+    const getTeamsMeetingLink = jest.spyOn(GetMeetingLink, 'getTeamsMeetingLink');
+    getTeamsMeetingLink.mockImplementation(() => {
+      return {
+        meetingLink:
+          '?meetingURL=https%3A%2F%2Fteams.microsoft.com%2Fl%2Fmeetup-join%2F19%253ameeting_AAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA%2540thread.v2%2F0%3Fcontext%3D%257b%2522Tid%2522%253a%252200000000-0000-0000-0000-000000000000%2522%252c%2522Oid%2522%253a%252200000000-0000-0000-0000-000000000000%2522%257d'
+      } as TeamsMeetingLinkLocator;
+    });
     const visit = mount(<Visit />);
 
     await runFakeTimers();
 
     visit.update();
 
     const spinners = visit.find(Spinner);
-    const joinMeetings = visit.find(JoinTeamsMeeting);
+    const teamsMeeting = visit.find(TeamsMeeting);
```

Comment:
Same here, find the JoinTeamsMeeting and RoomsMeeting and assert that they are not rendered